### PR TITLE
Editor Script to convert TMP to RTLTMP

### PR DESCRIPTION
**Pull request by BenIVRE**
Also can convert Text to RTLTMP.

Taken from a PR from the base repo: https://github.com/pnarimani/RTLTMPro/pull/114

### CODE REVIEW

**Review by RicardoAlmeida-IVRE — APPROVED** [review]

**Review by barry-ivre — APPROVED** [review]

File: Assets/RTLTMPro/Scripts/Editor/ConvertToTextMeshPro.cs [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+        [MenuItem("CONTEXT/TextMeshProUGUI/Convert to RTLTMPro")]
+        public static void ConvertTextMeshProUGUI(MenuCommand command)
+        {
+            var text = (TextMeshProUGUI)command.context;
+            Convert(text);
+        }
+        
+        /// <summary>
+        /// Unfortunately I'm not sure how to do GUID swapping, so references to the text component will break, but at least it should make the transition much safer and quicker
```
Comment (ruben-ivre) [inline Assets/RTLTMPro/Scripts/Editor/ConvertToTextMeshPro.cs:117]: I thought this asset handled the conversion to TMPro without losing references. Breaking existing references is not an option, there are thousands.

Comment (BenIVRE) [reply Assets/RTLTMPro/Scripts/Editor/ConvertToTextMeshPro.cs:117]: That's not what this asset does. We've already handled that in https://github.com/engage-xr/Engage_Main/pull/2004

Comment (BenIVRE) [reply Assets/RTLTMPro/Scripts/Editor/ConvertToTextMeshPro.cs:117]: You do raise a valid point though, if we go from TMP -> RTLTMP we would still potentially lose some references.

Comment (ruben-ivre) [reply Assets/RTLTMPro/Scripts/Editor/ConvertToTextMeshPro.cs:117]: I meant references in other components, #2004 only handled references in code, but not from other non-code components.

**Comment by BenIVRE** [comment]
Closing this as we can handle it from the New UI Widgets converter by changing the type it goes to to be RTLTMPro

**Pull request closed without merging**